WowPacketParser/Store/Objects/GameObjectModel.cs

```diff
@@ -16,25 +16,27 @@ public sealed class GameObjectModel : IDataModel
         [DBFieldName("map")]
         public uint? Map;
 
-        [DBFieldName("zoneId")]
+        [DBFieldName("zone_id")]
         public uint? ZoneID;
 
-        [DBFieldName("areaId")]
+        [DBFieldName("area_id")]
         public uint? AreaID;
 
-        [DBFieldName("spawnMask", TargetedDatabase.Zero, TargetedDatabase.Legion)]
+        [DBFieldName("spawn_mask", TargetedDatabase.Zero, TargetedDatabase.Legion)]
         public uint? SpawnMask;
 
-        [DBFieldName("spawnDifficulties", TargetedDatabase.Legion)]
+        [DBFieldName("spawn_difficulties", TargetedDatabase.Legion)]
         public string spawnDifficulties;
 
-        [DBFieldName("phaseMask", TargetedDatabase.WrathOfTheLichKing, TargetedDatabase.Cataclysm)]
+        [DBFieldName("phase_mask", TargetedDatabase.Classic, TargetedDatabase.Zero)]
+        [DBFieldName("phase_mask", TargetedDatabase.WrathOfTheLichKing, TargetedDatabase.Cataclysm)]
         public uint? PhaseMask;
 
-        [DBFieldName("PhaseId", TargetedDatabase.Cataclysm)]
+        [DBFieldName("phase_id", TargetedDatabase.Cataclysm)]
         public string PhaseID;
 
-        [DBFieldName("PhaseGroup", TargetedDatabase.Cataclysm)]
+        [DBFieldName("phase_mask", TargetedDatabase.Classic, TargetedDatabase.Zero)]
```

Comment:
Does classic event have phased content? Even if it does, since its based on legion client, shouldnt it have the modern fields (id/group/flags)?

Reply:
This I was wondering myself and in essence it doesnt. Just same tech.

Reply:
That file looks different that what I have on my fork. The `gameobject` table has no fields related to phases that are exported in the sql when parsing classic sniffs.

```
[DBFieldName("phase_mask", TargetedDatabase.Zero, TargetedDatabase.Cataclysm)]
public uint? PhaseMask;
```

Reply:
Changed during TBC impl for db save.